library/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/utils/Base64Utils.kt

```diff
@@ -0,0 +1,17 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+package com.datadog.android.sessionreplay.internal.utils
+
+import android.util.Base64
+import androidx.annotation.WorkerThread
+import java.io.ByteArrayOutputStream
+
+internal object Base64Utils {
+    @WorkerThread
+    internal fun serializeToBase64String(byteArrayOutputStream: ByteArrayOutputStream): String =
+        Base64.encodeToString(byteArrayOutputStream.toByteArray(), Base64.DEFAULT)
```

Comment:
again have you checked if this is throwing something under the hood ?

Reply:
see prev comment about the wrapper classes - there are now wrappers for base64, bitmap and canvas